snippets/mask-blobs.liquid

```diff
@@ -0,0 +1,12 @@
+{% comment %}
+  Renders blob shapes used as image masks.
+{% endcomment %}
+{% style %}
```

Comment:
Any reason this needs to be a liquid snippet and not a CSS asset file? As a snippet it will be rendered with each section that uses it, which could be several times (e.g. if the template has an image-with-text section followed by featured-collection. Whereas if you have it as a CSS file it will be downloaded once. I believe if there are multiple `<link rel="stylesheet"` to the same stylesheet the subsequent ones get ignored.